src/jet/elements/environment.rs

```diff
@@ -70,7 +70,7 @@ pub struct ElementsEnv {
     /// Optional Annex.
     annex: Option<Vec<u8>>,
     /// Genesis block hash
-    genesis_hash: elements::BlockHash,
+    genesis_hash: BlockHash,
```

Comment:
In 66c07d2ce8474e20b9bc66a566f654c6b8c5f3b4:

Please use the `elements::` or `bitcoin::` prefix for blocks, transactions, and any related types.

Reply:
I think I had conflicting versions of elements at some point. This line even didn't need to change.